packages/fungible-token/bridge-fungible-token/proxy/src/proxy.sw

```diff
@@ -0,0 +1,153 @@
+contract;
+
+use std::{
+    call_frames::first_param,
+    constants::ZERO_B256,
+    execution::run_external,
+    hash::Hash,
+    hash::sha256,
+    string::String,
+};
+use standards::{src14::SRC14, src20::SRC20, src5::{AccessError, SRC5, State}};
+use contract_message_receiver::MessageReceiver;
+use interface::{bridge::Bridge, src7::{Metadata, SRC7}};
```

Comment:
Why is src7 imported from interface and not standards?

Reply:
Cannot remember the exact reason except that it had to do with the pending PR https://github.com/FuelLabs/fuel-bridge/pull/103, something about having compilation or SDK problems due to the encoding. It has been a long standing task to implement SRC-8 since the new encoding is available now